storage/simulation/store.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Bram Gruneir (bram+code@cockroachlabs.com)
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"sync"
+
+	"github.com/cockroachdb/cockroach/proto"
+)
+
+const (
+	bytesPerRange    = 64 * (int64(1) << 20) // 64 MiB
```

Comment:
Untyped constants will be automatically typed when used in an assignment. That's why you can do `desc.Capacity = 1`.